pykern/pkdebug.py

```diff
@@ -40,10 +40,9 @@
         pkdp('user entered: {}', val)
         pkdc('user context: name={name}, id={id}', **user_rec)
 
-    If you do nothing, the first print statement would come
-    out always. The second wouldn't come out unless you
-    specified a ""control" via the environment variable
-    ``$PYKERN_PKDEBUG_CONTROL``:
+    The first print statement always appears.
+    The second does not unless you specify a "control" via the
+    environment variable ``$PYKERN_PKDEBUG_CONTROL``:
 
         PYKERN_PKDEBUG_CONTROL=my_mod python my_prog.py
```

Comment:
There is a shortcut for this value:

`        pkdebug=my_mod python my_prog.py`

Document both.